internal/gitops/structure/templates/organization/template.go

```diff
@@ -0,0 +1,15 @@
+package organization
+
+import (
+	_ "embed"
+)
+
+//go:embed organization.yaml.tmpl
+var organization string
+
+// OrganizationDirectoryTemplate returns organization directory layout.
+func OrganizationDirectoryTemplate() []string {
```

Comment:
Same as for management clusters. Also maybe it is a thing in go, but I miss the verb from function names.